Index: golang-github-containers-common/libimage/manifests/manifests.go
===================================================================
--- golang-github-containers-common.orig/libimage/manifests/manifests.go
+++ golang-github-containers-common/libimage/manifests/manifests.go
@@ -16,7 +16,6 @@ import (
 	"github.com/containers/image/v5/manifest"
 	"github.com/containers/image/v5/pkg/compression"
 	"github.com/containers/image/v5/signature"
-	"github.com/containers/image/v5/signature/signer"
 	is "github.com/containers/image/v5/storage"
 	"github.com/containers/image/v5/transports"
 	"github.com/containers/image/v5/transports/alltransports"
@@ -63,7 +62,7 @@ type PushOptions struct {
 	ImageListSelection               cp.ImageListSelection // set to either CopySystemImage, CopyAllImages, or CopySpecificImages
 	Instances                        []digest.Digest       // instances to copy if ImageListSelection == CopySpecificImages
 	ReportWriter                     io.Writer             // will be used to log the writing of the list and any blobs
-	Signers                          []*signer.Signer      // if non-empty, asks for signatures to be added during the copy using the provided signers.
+	//Signers                          []*signer.Signer      // if non-empty, asks for signatures to be added during the copy using the provided signers.
 	SignBy                           string                // fingerprint of GPG key to use to sign images
 	SignPassphrase                   string                // passphrase to use when signing with the key ID from SignBy.
 	SignBySigstorePrivateKeyFile     string                // if non-empty, asks for a signature to be added during the copy, using a sigstore private key file at the provided path.
@@ -253,7 +252,7 @@ func (l *list) Push(ctx context.Context,
 		DestinationCtx:                   options.SystemContext,
 		ReportWriter:                     options.ReportWriter,
 		RemoveSignatures:                 options.RemoveSignatures,
-		Signers:                          options.Signers,
+		//		Signers:                          options.Signers,
 		SignBy:                           options.SignBy,
 		SignPassphrase:                   options.SignPassphrase,
 		SignBySigstorePrivateKeyFile:     options.SignBySigstorePrivateKeyFile,
Index: golang-github-containers-common/libimage/copier.go
===================================================================
--- golang-github-containers-common.orig/libimage/copier.go
+++ golang-github-containers-common/libimage/copier.go
@@ -17,7 +17,6 @@ import (
 	"github.com/containers/image/v5/docker/reference"
 	"github.com/containers/image/v5/pkg/compression"
 	"github.com/containers/image/v5/signature"
-	"github.com/containers/image/v5/signature/signer"
 	storageTransport "github.com/containers/image/v5/storage"
 	"github.com/containers/image/v5/types"
 	encconfig "github.com/containers/ocicrypt/config"
@@ -107,7 +106,7 @@ type CopyOptions struct {
 	SignaturePolicyPath string
 	// If non-empty, asks for signatures to be added during the copy
 	// using the provided signers.
-	Signers []*signer.Signer
+	//Signers []*signer.Signer
 	// If non-empty, asks for a signature to be added during the copy, and
 	// specifies a key ID.
 	SignBy string
@@ -312,7 +311,7 @@ func (r *Runtime) newCopier(options *Cop
 	c.imageCopyOptions.OciEncryptLayers = options.OciEncryptLayers
 	c.imageCopyOptions.OciDecryptConfig = options.OciDecryptConfig
 	c.imageCopyOptions.RemoveSignatures = options.RemoveSignatures
-	c.imageCopyOptions.Signers = options.Signers
+	//c.imageCopyOptions.Signers = options.Signers
 	c.imageCopyOptions.SignBy = options.SignBy
 	c.imageCopyOptions.SignPassphrase = options.SignPassphrase
 	c.imageCopyOptions.SignBySigstorePrivateKeyFile = options.SignBySigstorePrivateKeyFile
Index: golang-github-containers-common/libimage/manifest_list.go
===================================================================
--- golang-github-containers-common.orig/libimage/manifest_list.go
+++ golang-github-containers-common/libimage/manifest_list.go
@@ -433,7 +433,7 @@ func (m *ManifestList) Push(ctx context.
 		ImageListSelection:               options.ImageListSelection,
 		Instances:                        options.Instances,
 		ReportWriter:                     options.Writer,
-		Signers:                          options.Signers,
+		//Signers:                          options.Signers,
 		SignBy:                           options.SignBy,
 		SignPassphrase:                   options.SignPassphrase,
 		SignBySigstorePrivateKeyFile:     options.SignBySigstorePrivateKeyFile,
